### PR TITLE
feat: agency hyperlinks on RI and MA buoy pages

### PR DESCRIPTION
**Pull request by tdivoll**
Added agency hyperlinks to state buoy pages

### CODE REVIEW

File: src/views/datasets/ma-buoy/summary.vue [hunk 1]
```diff
@@ -4,13 +4,21 @@
...
         {{ store.maxDate.slice(0, 4) }} and was collected by the Narragansett
-        Bay Fixed Station Monitoring Network (NBFSMN) with MassDEP as the lead
-        agency. The heatmap below summarizes the number of observations
-        collected for each month for different variables. Use this heatmap to
-        help you decide what data you want to visualize or download. When you
-        have an idea, go ahead and select the buoys, variables and dates to
-        explore. Or download the data in the most appropriate format for your
-        analyses! To begin, select a variable to see what data is available.
+        Bay Fixed Station Monitoring Network (<a
+          href="http://www.dem.ri.gov/programs/emergencyresponse/bart/stations.php"
```
Comment (mcmcgrath13) [inline src/views/datasets/ma-buoy/summary.vue:8]: would it be better to put the link around the long form instead of the abbreviation?

Comment (mcmcgrath13) [reply src/views/datasets/ma-buoy/summary.vue:8]: or both?

File: src/views/datasets/ri-buoy/summary.vue [hunk 2]
```diff
@@ -3,15 +3,26 @@
...
-      summarizes the number of observations collected for each month for
-      different variables. Use this heatmap to help you decide what data you
-      want to visualize or download. When you have an idea, go ahead and select
-      the buoys, variables and dates to explore. Or download the data in the
-      most appropriate format for your analyses! To begin, select a variable to
-      see what data is available.
+      Fixed Station Monitoring Network (<a
+        href="http://www.dem.ri.gov/programs/emergencyresponse/bart/stations.php"
+        >NBFSMN</a
+      >) with <a href="http://www.dem.ri.gov/programs/water/">RIDEM-OWR</a> as
```
Comment (mcmcgrath13) [inline src/views/datasets/ri-buoy/summary.vue:9]: make all of these links `target="_blank" rel="noreferrer noopener"` so they open in a new tab (if we're feeling fancy, we could even pull that out into a mini-component, but might be overkill here)

File: src/views/datasets/ma-buoy/summary.vue (hunk 1 above)
Comment (tdivoll) [reply src/views/datasets/ma-buoy/summary.vue:8]: A lot of them only have an acronym, so maybe for consistency, just the acronyms?

Comment (mcmcgrath13) [reply src/views/datasets/ma-buoy/summary.vue:8]: I'd maybe do it around the whole thing `<a ...>Agency Name (AN)</a>

File: src/views/datasets/ri-buoy/summary.vue [hunk 3]
```diff
@@ -22,6 +54,8 @@
       The historical data available on this site has been compiled from the
       <a
         href="http://www.dem.ri.gov/programs/emergencyresponse/bart/stations.php"
+        target="_blank"
+        rel="noreferrer noopener"
         >Narragansett Bay Fixed-Site Monitoring Network</a
       >. See <a href="nbfsmn_disclaimer.pdf">the disclaimer</a> for more
```
Comment (tdivoll) [inline src/views/datasets/ri-buoy/summary.vue:60]: This link doesn't seem to work in the localhost version. Is the pdf only included in the deployed version?

File: src/views/datasets/fish/summary.vue [hunk 4]
```diff
@@ -56,20 +56,30 @@
...
+          rel="noreferrer noopener"
           >University of Rhode Island Graduate School of Oceanography Fish Trawl
           Survey</a
         >.
       </p>
       <p>
         <strong>To cite this data</strong>:
-        <a href="https://web.uri.edu/fishtrawl/data/">Fish Trawl Data Page</a>
+        <a
+          href="https://web.uri.edu/gso/research/fish-trawl/data/"
```
Comment (tdivoll) [inline src/views/datasets/fish/summary.vue:70]: Updated these links to reflect changes at `web.uri.edu`

File: src/views/datasets/ri-buoy/summary.vue (hunk 3 above)
Comment (mcmcgrath13) [reply src/views/datasets/ri-buoy/summary.vue:60]: just opened a PR to fix this #33

File: src/views/datasets/fish/summary.vue (hunk 4 above)
Comment (mcmcgrath13) [reply src/views/datasets/fish/summary.vue:70]: thanks!

File: src/views/datasets/ri-buoy/summary.vue (hunk 3 above)
Comment (tdivoll) [reply src/views/datasets/ri-buoy/summary.vue:60]: thank you!

**Pull request merged**